docs/contexts.md

```diff
@@ -43,7 +43,7 @@ The database for Hello.Repo has been created
 14:38:37.418 [info]  Already up
 ```
 
-Now we're ready to create our accounts context. We'll use the `phx.gen.html` task which creates a context module that wraps up Ecto access for creating, updating, and deleting users, along with web files like controllers and templates for the web-interface into our context. Run the following command at your project root:
+Now we're ready to create our accounts context. We'll use the `phx.gen.html` task which creates a context module that wraps up Ecto access for creating, updating, and deleting users, along with web files like controllers and templates for the Web interface into our context. Run the following command at your project root:
```

Comment:
I think we want a lower case 'w' here please. This reflects usage throughout the rest of the guides.